### PR TITLE
Unifi commit flow report naming

### PR DESCRIPTION
**Pull request by vulder**
related se-passau/VaRA#507

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/experiments/commit_report_experiment.py  6
         

Clones added
============
- varats/experiments/commit_report_experiment.py  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA-Tool-Suite/VaRA-Tool-Suite/pullRequest?prid=4507556&bid=15191354)

File: varats/experiments/commit_report_experiment.py [hunk 1]
```diff
@@ -124,14 +125,13 @@ def analyze(self) -> actions.StepResult:
                     timeout_duration))
 
 
-class GitBlameAnnotationReport(VersionExperiment):
+class CommitReportExperiment(VersionExperiment):
     """
     Generates a commit flow report (CFR) of the project(s) specified in the
```
Comment (boehmseb) [inline varats/experiments/commit_report_experiment.py]: You are renaming this to _commit report_ right?

```suggestion
    Generates a commit report (CR) of the project(s) specified in the
```

File: README.md [hunk 2]
```diff
@@ -101,10 +101,10 @@ vara-gen-bbconfig
 ```
 
 ### Running BenchBuild experiments
-Second, we change into the benchbuild folder and run an experiment like `GitBlameAnnotationReport` on provided projects, in this case we use `gzip`.
+Second, we change into the benchbuild folder and run an experiment like `CommitReportExperiment` on provided projects, in this case we use `gzip`.
 ```console
 cd $VARA_ROOT/benchbuild
-benchbuild -vv run -E GitBlameAnnotationReport gzip
+benchbuild -vv run -E GenerateCommitReport gzip
```
Comment (boehmseb) [inline README.md:107]: Why was both called _Report_ before and now there is a distinction between _Experiment_ and _Report_?
What should the naming scheme be in the end?

File: varats/experiments/commit_report_experiment.py [hunk 3]
```diff
@@ -124,14 +125,13 @@ def analyze(self) -> actions.StepResult:
...
     call.
     """
 
-    NAME = "GitBlameAnnotationReport"
-
+    NAME = "GenerateCommitReport"
```
Comment (boehmseb) [inline varats/experiments/commit_report_experiment.py:134]: Same as in readme: Why _Report_ vs _Experiment?

File: README.md [hunk 4]
```diff
@@ -101,10 +101,10 @@ vara-gen-bbconfig
 ```
 
 ### Running BenchBuild experiments
-Second, we change into the benchbuild folder and run an experiment like `GitBlameAnnotationReport` on provided projects, in this case we use `gzip`.
+Second, we change into the benchbuild folder and run an experiment like `CommitReportExperiment` on provided projects, in this case we use `gzip`.
```
Comment (vulder) [inline README.md]:
```suggestion
Second, we change into the benchbuild folder and run an experiment that generates `CommitReports` for provided projects, in this case we use `gzip`.
```

**Review by boehmseb — APPROVED** [review]
lgtm

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/experiments/commit_report_experiment.py  6
         

Clones added
============
- varats/experiments/commit_report_experiment.py  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA-Tool-Suite/VaRA-Tool-Suite/pullRequest?prid=4507556&bid=15191354)

**Pull request merged**